protocol/contracts/beanstalk/silo/ConvertFacet.sol

```diff
@@ -74,37 +74,54 @@ contract ConvertFacet is ReentrancyGuard {
         nonReentrant
         returns (int96 toStem, uint256 fromAmount, uint256 toAmount, uint256 fromBdv, uint256 toBdv)
     {
-        address toToken; address fromToken; uint256 grownStalk; address account;
+        address toToken; address fromToken; uint256 grownStalk; address account; bool decreaseBDV;
+
+        /** @dev account and decreaseBDV are initialized at the start of LibConvert.convert()
```

Comment:
comment is good, but imo not needed as line 87 is self explanatory 
- could move to LibConvert to inform future developers.

Reply:
agree on this one moved [here](https://github.com/BeanstalkFarms/Beanstalk/pull/736/commits/10fc00c5eba6b41bb6e26b1eb8b020f7b458478d) as a note